app/period-subdomain/src/main/java/com/intellistart/interviewplanning/domain/model/Period.kt

```diff
@@ -1,11 +1,9 @@
-package intellistart.interviewplanning.model.period
+package com.intellistart.interviewplanning.domain.model
```

Comment:
is this data class saved to redis? I don't expect any json annotations on domain object